raiden_installer/web.py

```diff
@@ -554,6 +555,27 @@ def get(self, configuration_file_name):
             }
         )
 
+    def post(self, configuration_file_name):
+        configuration_file = RaidenConfigurationFile.get_by_filename(configuration_file_name)
+        account = configuration_file.account
+        w3 = make_web3_provider(configuration_file.ethereum_client_rpc_endpoint, account)
+        ex_currency_amt = json_decode(self.request.body)
```

Comment:
does the json_decode by tornado something specific or why not using the json package?

Reply:
Nothing specific just that I observed it being used in tornado code at many places hence decided to use it.